superchain/configs/sepolia-dev-0/base-devnet-0.toml

```diff
@@ -10,9 +10,12 @@ canyon_time = 1698436800 # Fri 27 Oct 2023 20:00:00 UTC
 delta_time = 1706555000 # Mon 29 Jan 2024 19:03:20 UTC
 ecotone_time = 1706634000 # Tue 30 Jan 2024 17:00:00 UTC
 fjord_time = 1715961600 # Fri 17 May 2024 16:00:00 UTC
+granite_time = 1723046400 # Wed 7 Aug 2024 16:00:00 UTC
```

Comment:
I think this is probably an unintended change. The technique of overwriting the toml using the Go bindings (themselves mostly the result of unmarshalling the toml) has the effect of picking up hardfork times from superchain.toml and dumping them in here.

Reply:
Same for a few other files, too.

Reply:
Are you saying the `granite_time` should not be added at this time? Or it will never be added? IMO its better to explicitly set the timestamp in a chain's config file instead of relying on downstream software to configure the chain for you. Its more difficult to understand when a chain's hardfork will activate (or was activated) if its not part of its config file